docker/base/Dockerfile

```diff
@@ -142,6 +142,6 @@ RUN locale-gen en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV PYTHONIOENCODING UTF-8
 
-COPY setup_common.sh setup_clusterfuzz.sh start_clusterfuzz.sh setup_mock_metadata.sh start.sh /data/
+COPY setup_common.sh setup_clusterfuzz.sh start_clusterfuzz.sh setup_mock_metadata.sh start.sh Pipfile Pipfile.lock /data/
```

Comment:
Do we need to add pipefiles here to install new packages?

Reply:
If we add it to src/Pipfile, we don't need to install it in the base image.